doc/releases/changelog-dev.md

```diff
@@ -152,6 +152,10 @@
 
 <h3>Improvements</h3>
 
+* `qml.matrix` now can also compute the matrix of tapes/QNodes that contain multiple
+  broadcasted operations, as they are present in decompositions of broadcasted operations.
```

Comment:
> as they are present in decompositions of broadcasted operations.

This reads a bit confusing, what does this refer to?

Reply:
Ah, agreed :D It refers to this problem/application popping up when we do
```python
with qml.tape.QuantumTape() as tape:
    qml.IsingXX(np.array([0.4, 2.1, -0.4]), wires=[1, 2]).decomposition()
qml.matrix(tape)
```
i.e. when we decompose a broadcasted operation and then want to compute the matrix of the resulting tape.

Reply:
Tried to make it more explicit/clear

Reply:
Thanks! Left a suggestion.